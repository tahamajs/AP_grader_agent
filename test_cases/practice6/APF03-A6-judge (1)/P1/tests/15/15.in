POST signup ? username "amir" password "rfz"
POST login ? username "amir" password "rfz"
PUt my_district ? district "aliabad"
PUT mydistrict ? district "aliabad"
PUT my_district ? district "aliabad"
PUT my_district ? district "Karegar"