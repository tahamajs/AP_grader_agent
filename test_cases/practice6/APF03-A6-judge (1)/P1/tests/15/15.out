OK
Permission Denied
Bad Request
Not Found
Not Found
OK